ormx-macros/src/lib.rs

```diff
@@ -63,6 +63,9 @@ mod utils;
 /// **`#[ormx(get_many)]`**:  
 /// `{pub} async fn get_by_{field_name}(&{field_type}) -> Result<Vec<Self>>`
 ///
+/// **`#[ormx(get_any)]`**:  
+/// `{pub} async fn get_by_{field_name}(&[{field_type}]) -> Result<Vec<Self>>`
```

Comment:
I am wondering if `get_by_any` would be a better name? The default name of the method would become `get_by_any_{field_name}`.

`get_by_any_email(&[String])` is clearer than `get_by_email(&[String])` in my opinion

Reply:
I agree! 
Been inactive here for a while, sry. Do you still want to change this? I think get_by_any_* is a far better name. As soon as you are ready, I can merge this.

Reply:
No problem :) will push an update somewhere this week